source/MaterialXRenderGlsl/GlslRenderer.cpp

```diff
@@ -70,6 +70,11 @@ void GlslRenderer::initialize()
     }
 }
 
+void GlslRenderer::initialize()
+{
+    initialize(nullptr);
+}
+
```

Comment:
@ashwinbhat Instead of maintaining two separate `initialize` methods, perhaps this could be a single `initialize` method with a default value for its `sharedContextHandle` argument?  That pattern should extend to Python as well, as PyBind11 allows you to specify default values for arguments.